stripper/ze_huggy_wuggy_chap4_halloween_rus4.cfg

```diff
@@ -0,0 +1,17 @@
+;prevent zombies getting boosted into next room by teleporting them away and moving the trigger a bit back
+modify:
+{
+	match:
+	{
+		"classname" "trigger_push"
```

Comment:
Leaving this push functional doesn't seem like a very good idea if you're going to immediately tp them, could lead to weird boosts.